backends/platform/wince/CEDevice.h

```diff
@@ -23,6 +23,8 @@
 #ifndef CEDEVICE_H
 #define CEDEVICE_H
 
+#include <windows.h>
```

Comment:
You’ve already included this in the .cpp file. Is it also needed here?

Reply:
Fixed.